deemix/src/utils/crypto.ts

```diff
@@ -1,80 +1,110 @@
-import {
-	createCipheriv,
-	createHash,
-	createDecipheriv,
-	getCiphers,
-} from "crypto";
+import * as crypto from "crypto";
```

Comment:
We should import individual functions if we can, since this will import the entire crypto library (even the parts we don't use)